### PR TITLE
fix #8

### LINKED ISSUE #8: Add way to drop in the widget to predefined location on screen
The original widget places itself on screen (see Placement: ... selector <https://helpukrainewinwidget.org/install-manually>).

We need to replicate that so that the Widget would be easier to integrate into the app.

### PR DESCRIPTION
**Pull request by solid-danylosafonov**

### CODE REVIEW

**Comment by solid-danylosafonov** [comment]
Small overflow in example
image

- [x] Fix overflow
- [ ] Also probably need to add overlaywidget HelpWidget (duplicate alignment)
- [x] And I want to add knobs for alignment to example

**Comment by solid-danylosafonov** [comment]
unsure about last commit: something like that will be needed for all variations, so we either write it twice (four times), or some kind of interface is needed to not rewrite that helper widget, instead we would do something like:
```
class OverlayHelpWidget<T implements IHelpWidget> extends StatelessWidget {
  final Alignment alignment;
  final Widget background;
  final HorizontalHelpWidget Function(Alignment) factory;

  const OverlayHelpWidget({
    required this.alignment,
    required this.background,
    required this.factory,
  });

  @override
  Widget build(BuildContext context) {
    return OverlayWidget(
      alignment: alignment,
      overlayWidget: factory(
        alignment: alignment,
      ),
      child: background,
    );
  }
}
```

and then
`
OverlayHelpWidget(HorizontalHelpWidget, alignment)
`

also other variations [will] have multiple duplicate field

**Comment by solid-danylosafonov** [comment]
> and then `OverlayHelpWidget(HorizontalHelpWidget, alignment)`
> 
> also other variations [will] have multiple duplicate field

nvm that's probably too meta and won't be possible in dart (or it won't be pretty)

**Comment by solid-yuriiprykhodko** [comment]
@solid-danylosafonov Made a few changes that seem to fix the double `alignment` arg issue.

**Comment by solid-danylosafonov** [comment]
image

image

image

File: example/lib/storybook/stories/help_widget_story.dart [hunk 1]
```diff
@@ -4,17 +4,61 @@ import 'package:widgetbook/widgetbook.dart';
...
+                padding: EdgeInsets.zero,
+                backgroundColor: Colors.black,
+                height: testHeight,
+                width: testWidth,
+                child: OverlayWidget(
+                  alignment: _alignment,
+                  overlayWidget: HorizontalHelpWidget(),
+                  child: ListView(
+                    children: [
+                      for (var i = 0; i < _listLength; i++)
```
Comment (solid-yuriiprykhodko) [inline example/lib/storybook/stories/help_widget_story.dart]: List.generate?

**Review by solid-yuriiprykhodko — CHANGES_REQUESTED** [review]
A small refactoring nitpick.

File: example/lib/storybook/stories/help_widget_story.dart (hunk 1 above)
Comment (solid-danylosafonov) [reply example/lib/storybook/stories/help_widget_story.dart]: my bad

**Review by solid-yuriiprykhodko — APPROVED** [review]
LGTM!

**Pull request merged**